plugins/ui/zen-mode.nix

```diff
@@ -0,0 +1,171 @@
+            - a percentage of the width / height of the editor when <= 1
+            - a function that returns the width or the height
+          '';
+
+        height =
+          helpers.defaultNullOpts.mkNullable
+          (
+            with helpers.nixvimTypes;
+              oneOf [
+                ints.positive
+                (numbers.between 0.0 1.0)
+                rawLua
+              ]
+          )
+          "1"
+          ''
+            Height of the Zen window.
+
+            Can be:
+            - an absolute number of cells when > 1
+            - a percentage of the width / height of the editor when <= 1
+            - a function that returns the width or the height
+          '';
+
+        options =
+          helpers.defaultNullOpts.mkAttrsOf types.anything "{}"
+          ''
+            By default, no options are changed for the Zen window.
+            You can set any `vim.wo` option here.
+
+            Example:
+            ```nix
+              {
+                signcolumn = "no";
+                number = false;
+                relativenumber = false;
+                cursorline = false;
+                cursorcolumn = false;
+                foldcolumn = "0";
+                list = false;
+              }
+            ```
+          '';
+      };
+      plugins = {
+        options =
+          helpers.defaultNullOpts.mkAttrsOf types.anything
+          ''
+            {
+              enabled = true;
+              ruler = false;
+              showcmd = false;
+              laststatus = 0;
+            }
+          ''
+          ''
+            Disable some global vim options (`vim.o`...).
+          '';
+
+        #    twilight = { enabled = true }, -- enable to start Twilight when zen mode opens
+        #    gitsigns = { enabled = false }, -- disables git signs
+        #    tmux = { enabled = false }, -- disables the tmux statusline
+        #    -- this will change the font size on kitty when in zen mode
+        #    -- to make this work, you need to set the following kitty options:
+        #    -- - allow_remote_control socket-only
+        #    -- - listen_on unix:/tmp/kitty
+        #    kitty = {
+        #      enabled = false,
+        #      font = "+4", -- font size increment
+        #    },
+        #    -- this will change the font size on alacritty when in zen mode
+        #    -- requires  Alacritty Version 0.10.0 or higher
+        #    -- uses `alacritty msg` subcommand to change font size
+        #    alacritty = {
+        #      enabled = false,
+        #      font = "14", -- font size
+        #    },
+        #    -- this will change the font size on wezterm when in zen mode
+        #    -- See also the Plugins/Wezterm section in this projects README
+        #    wezterm = {
+        #      enabled = false,
+        #      -- can be either an absolute font size or the number of incremental steps
+        #      font = "+4", -- (10% increase per step)
+        #    },
```

Comment:
Why have this commented?

Reply:
It was a mistake. I won't explicitly those options.